tasklist/qa/integration-tests/src/test/java/io/camunda/tasklist/util/TestApplication.java

```diff
@@ -38,7 +41,11 @@
 @Import({
   WebappsModuleConfiguration.class,
   CommonsModuleConfiguration.class,
-  TestTasklistPropertiesOverride.class
+  TestTasklistPropertiesOverride.class,
+  // Unified Configuration classes
+  UnifiedConfiguration.class,
+  UnifiedConfigurationHelper.class,
+  GatewayBasedPropertiesOverride.class,
```

Comment:
:question: I wonder why Gateway properties are needed here. I think it is because it is scanning the common module. It is ok for this PR. But we should try to remove this dependency if this is not used.